contrail_api_cli_extra/clean/refs.py

```diff
@@ -69,10 +71,22 @@ def _remove_refs(self, paths):
         printo("[%s -> %s] deleted" % (source, target))
         self._remove_refs(paths[2:])
 
-    def __call__(self, paths=None, ref_type=None, target_type=None, cassandra_servers=None, **kwargs):
+    def _read_file(self, resources_file):
```

Comment:
There is a python `readLines` function that could do this for you

Reply:
There are the "\n" for some item I don't know if it could imply some errors after in the code ?